src/main/java/com/hcl/appscan/jenkins/plugin/scanners/ScannerFactory.java

```diff
@@ -23,6 +23,9 @@ public static Scanner getScanner(String type, String target) {
 		case STATIC_ANALYZER:
 			ret = new StaticAnalyzer(target);
 			break;
+            	case SOFTWARE_COMPOSITION_ANALYSIS:
```

Comment:
A minor point, but I suggest making this constant SOFTWARE_COMPOSITION_ANALYZER to be in line with the others.

Reply:
Okay, I will make it ANALYZER.